### PR TITLE
ADEN-4133 Load Ooyala player from creative template

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: extensions/wikia/VideoHandlers/js/handlers/Ooyala.js [hunk 1]
```diff
@@ -72,6 +72,10 @@ define('wikia.videohandler.ooyala', [
 				}
 			});
 
+			if (typeof params.onCreate === 'function') {
+				params.onCreate(player);
+			}
+
 			// Log all events and values (for debugging)
 			/*messageBus.subscribe('*', 'tracking', function(eventName, payload) {
```
Comment (WikiaTech) [inline extensions/wikia/VideoHandlers/js/handlers/Ooyala.js]: MAJOR Remove this commented out code. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ACommentedCode)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: extensions/wikia/AdEngine/js/template/ooyala.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/*global define*/
+define('ext.wikia.adEngine.template.ooyala', [
+	'ext.wikia.adEngine.video.vastBuilder',
+	'wikia.log',
+	'wikia.videoBootstrap'
+], function (vastBuilder, log, VideoBootstrap) {
+	'use strict';
+	var libraryUrl = 'http://player.ooyala.com/v3/52bc289bedc847e3aa8eb2b347644f68?platform=html5-priority',
```
Comment (bognix) [inline extensions/wikia/AdEngine/js/template/ooyala.js:8]: this ID in URL is not something that we should keep private?

File: extensions/wikia/VideoHandlers/js/handlers/Ooyala.js (hunk 1 above)
Comment (bognix) [inline extensions/wikia/VideoHandlers/js/handlers/Ooyala.js]: I would follow the WikiaTech advice

**Review by bognix — APPROVED** [review]
LGTM, one question, one minor comment

File: extensions/wikia/AdEngine/js/template/ooyala.js (hunk 2 above)
Comment (fraszczakszymon) [reply extensions/wikia/AdEngine/js/template/ooyala.js:8]: It's being sent in request for Ooyala player - it's accessible anyway 👍

**Comment by bognix** [comment]
LGTM

**Comment by bognix** [comment]
Tested, merging.

**Pull request merged**